### ISSUE TITLE
Add email notifications when a new post / comment is created

### ISSUE THREAD

**Issue by mpscholten** [digitallyinduced/ihp-forum#8]

**Comment by themmes** [comment]
A daily summary (morning or evening) might be preferable. Does this relate to https://github.com/digitallyinduced/ihp/issues/303 ?

**Comment by mpscholten** [comment]
Good idea with the daily summary :) in that case we could just use a Script which we run daily via a cronjob. Would be a lot simpler than using a task queue IMO
